frontend/src/v5/services/intl.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { messagesEs } from '@/locales/messagesEs';
+import { createIntl, createIntlCache, IntlShape } from 'react-intl';
+
+let intlInternal:IntlShape = null;
+
+const DEFAULT_LOCALE = 'en';
+
+export const initializeIntl = (locale: string) => {
+	let messages = {};
+
+	switch (locale.split('-')[0]) {
```

Comment:
```suggestion
	switch (locale.toLowerCase().split('-')[0]) {
```

It looks like the language part that we're interested in should always be lowercase anyway and the region subtag should be uppercase, but should we ensure it is always lowercase for the comparison?